conda/recipes/rmm/recipe.yaml

```diff
@@ -0,0 +1,92 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2019-2025, NVIDIA CORPORATION.
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  head_rev: ${{ git.head_rev( "." )[:8] }}
+
+package:
+  name: rmm
+  version: ${{ version }}
+
+source:
+  - path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
+  script:
+    file: build.sh
+    secrets:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+
+    env:
+      CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER", default="sccache") }}
+      CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER", default="sccache") }}
+      CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER", default="sccache") }}
+      CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR", default="Ninja") }}
+      SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET", default="") }}
+      SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT", default="") }}
+      SCCACHE_REGION: ${{ env.get("SCCACHE_REGION", default="") }}
+      SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL", default="") }}
+      SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS", default="") }}
+      SCCACHE_S3_KEY_PREFIX: rmm-${{ env.get("RUNNER_ARCH", default="X64") | replace("X64", "linux64") | replace("ARM64", "aarch64") | lower }}
+
+requirements:
+  build:
+    - cmake ${{ cmake_version }}
+    - ninja
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("cuda") }}
+    - cuda-version =${{ cuda_version }}
+    - ${{ stdlib("c") }}
+  host:
+    - cuda-version =${{ cuda_version }}
+    - if: cuda_major == "11"
+      then:
+        - cudatoolkit
+        - cuda-python >=11.8.5,<12.0a0
+      else:
+        - cuda-cudart-dev
+        - cuda-python >=12.6.2,<13.0a0
+    - cython >=3.0.0
+    - rapids-build-backend >=0.3.0,<0.4.0.dev0
+    - librmm =${{ version }}
+    - python <3.13
+    - pip
+    - scikit-build-core >=0.10.0
+  run:
+    - if: cuda_major == "11"
+      then:
+        - cudatoolkit
+        - cuda-python >=11.8.5,<12.0a0
+      else:
+        - cuda-cudart
+        - cuda-python >=12.6.2,<13.0a0
+    - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+    - numba >=0.59.1,<0.61.0a0
```

Comment:
I missed removing this dependency in https://github.com/rapidsai/rmm/pull/1761.

Numba is no longer a runtime dependency of `rmm`.
```suggestion
```